Functions/AddRecord.dart

```diff
@@ -0,0 +1,150 @@
+import 'dart:io';
```

Comment:
keep package name, files name in small snake (separated by underscore) case in dart.
example: `functions/add_record.dart`